bitrise.yml

```diff
@@ -1418,6 +1418,44 @@ workflows:
       bitrise.io:
         stack: osx-xcode-14.1.x-ventura
         machine_type_id: g2.8core
+perfherder-test:
```

Comment:
The indentation is still wrong, it should be like the other exisiting workflows: 
https://github.com/mozilla-mobile/firefox-ios/blob/e1a9646678b9d64105d6617a313ac24193b927e0/bitrise.yml#L1323